ui/pages/home/beta/beta-home-footer.component.js

```diff
@@ -28,7 +29,7 @@ const BetaHomeFooter = () => {
               category: MetaMetricsEventCategory.Footer,
               event: MetaMetricsEventName.SupportLinkClicked,
               properties: {
-                url: SUPPORT_REQUEST_LINK,
+                url: SUPPORT_LINK,
```

Comment:
We should keep this one as `SUPPORT_REQUEST_LINK` though, since we're going to revert those change later after the internal beta 😅.

```suggestion
                url: SUPPORT_REQUEST_LINK,
```